test/yargs-parser.js

```diff
@@ -2233,6 +2197,44 @@ describe('yargs-parser', function () {
         })
       })
     })
+
+    describe('populate--', function () {
+      it('should populate "_" if "populate-- false', function () {
+        var result = parser([
+          'bare',
+          '--', '-h', 'eek', '--'
+        ], {
+          configuration: {'populate--': false}
+        })
+        result.should.have.property('_').and.deep.equal(['bare', '-h', 'eek', '--'])
+        result.should.not.have.property('--')
+      })
+
+      it('should populate the "--" array by default', function () {
+        var result = parser([
+          '--name=meowmers', 'bare', '-cats', 'woo', 'moxy',
+          '-h', 'awesome', '--multi=quux',
+          '--key', 'value',
+          '-b', '--bool', '--no-meep', '--multi=baz',
+          '--', '--not-a-flag', '-', '-h', '-multi', '--', 'eek'
+        ], {
+          '--': true
```

Comment:
whoops, I should actually just remove this configuration; since it defaults to `true` now.